Fix part of #14419: Refactor FILTER_FIELD_NAMES Enum class to Pascal case and values to UPPERCASE

Akshaj000 opened the pull request:
## Overview

1. This PR fixes or fixes part of  #14419 .
2. This PR does the following: Refactor FILTER_FIELD_NAMES Enum class to Pascal case and values to UPPERCASE

## Essential Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The linter/Karma presubmit checks have passed locally on your machine.
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
  - This lets reviewers restart your CircleCI tests for you.
- [x] The PR is made from a branch that's **not** called "develop".

## Proof that changes are correct

https://user-images.githubusercontent.com/83636572/155549837-d153a9ff-033a-42d9-85b9-aee59013746a.mp4


## PR Pointers

- Make sure to follow the [instructions for making a code change](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change).
- Oppiabot will notify you when you don't add a PR_CHANGELOG label. If you are unable to do so, please @-mention a code owner (who will be in the Reviewers list), or ask on [Gitter](https://gitter.im/oppia/oppia-chat).
- For what code owners will expect, see the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).
- Make sure your PR follows conventions in the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide), otherwise this will lead to review delays.
- Never force push. If you do, your PR will be closed.
- Oppiabot can assign anyone for review/help if you leave a comment like the following: "{{Question/comment}} @{{reviewer_username}} PTAL"
- Some of the e2e tests are flaky, and can fail for reasons unrelated to your PR. We are working on fixing this, but in the meantime, if you need to restart the tests, please check the ["If your build fails" wiki page](https://github.com/oppia/oppia/wiki/If-your-build-fails).

Akshaj000 committed:
app_feedback_report_constants.py:gae_models.py refractor FILTER_FIELD_NAMES to FilterFieldNames

Akshaj000 committed:
app_feedback_report_constants.py:gae_models.py change values to uppercase

Akshaj000 committed:
domain:app_feedback_report_constants.py refractor FILTER_FIELD_NAMES to FilterFieldNames

Akshaj000 committed:
domain:app_feedback_report_constants.py change values to uppercase

Akshaj000 committed:
domain:app_feedback_report_constants.py change values to uppercase

Akshaj000 committed:
domain:app_feedback_report_domain_test.py refractor FILTER_FIELD_NAMES to FilterFieldNames

Akshaj000 committed:
domain:app_feedback_report_domain_test.py change value platform to Uppercase

Akshaj000 committed:
domain:app_feedback_report_domain.py refractor FILTER_FIELD_NAMES to FilterFieldNames

Akshaj000 committed:
domain:app_feedback_report_services_test.py  refractor FILTER_FIELD_NAMES to FilterFieldNames

Akshaj000 committed:
storage/app_feedback_report:gae_models.py remove trailing whitespaces

Akshaj000 committed:
domain:app_feedback_report_constants.py add whitespace around operator

Akshaj000 committed:
storage/app_feedback_report:gae_models.py add two blank after function/class definition

Akshaj000 committed:
storage/app_feedback_report:gae_models.py add two blank after function/class definition

Akshaj000 committed:
app_feedback_report_constants.py:gae_models.py change values to uppercase

Akshaj000 committed:
domain:app_feedback_report_services_test.py  Change values to UPPERCASE

Akshaj000 committed:
domain:app_feedback_report_services_test.py Remove minor accidental change

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 commented:
Can u check this ? @JeeveshGarg PTAL

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

seanlip commented on core/storage/app_feedback_report/gae_models.py:
```diff
@@ -328,7 +321,7 @@ def get_all_unscrubbed_expiring_report_models(
+        cls, filter_field: FilterFieldNames
     ) -> List[str]:
         """Fetches values that can be used to filter reports by.
 
```
I think you forgot to update the typeinfo in the docstring below, but otherwise, everything looks good to me!

Akshaj000 replied:
Hi, i have changed that docstring. Can u check it out?

Akshaj000 replied:
Also ig i need a little help  on backend tests.

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 committed:
app_feedback_report:gae_models.py update typeinfo in the docstring

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 committed:
gae_models.py : change docstring

Akshaj000 commented:
Hi, I will try to fix the backend test soon. Sorry!

Akshaj000 committed:
Try fix backend test

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 commented:
> Hi, I will try to fix the backend test soon. Sorry!

![gnome-shell-screenshot-OQLPI1](https://user-images.githubusercontent.com/83636572/156712297-c6eeb9bf-5c88-4315-985d-ab2751d07d69.png)

I tried changing .name to .value. It solved errors in app feedback domain test and service test. But im getting this error in the models.

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 commented:
@krishita30j @seanlip  hey can u help me with this pr.? Can i know what im missing?

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

seanlip commented:
You might need to do a little debugging. Have you taken a look at the test code that is pointed to in the stacktrace to see what it does?

Akshaj000 committed:
gae_models_test.AppFeedbackReportModelTests change INVALID_FIELD.name to value

Akshaj000 committed:
Merge branch 'refract_FILTER_FIELD_NAMES' of https://github.com/Akshaj000/oppia into refract_FILTER_FIELD_NAMES

Akshaj000 commented:
> You might need to do a little debugging. Have you taken a look at the test code that is pointed to in the stacktrace to see what it does?

yes, i went through the test file,  ig i fixed the error. i hope its due to INVALID_FIELD.name i changed that to value, since in model it is used. @seanlip can u please check it now?

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES

Akshaj000 committed:
Merge branch 'oppia:develop' into refract_FILTER_FIELD_NAMES
